pkg/client/gen.go

```diff
@@ -156,35 +169,33 @@ func (*SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 
 	cfg.PluginEnvOverrides, plugins = applyK8sVersion(cfg.KubeVersion, cfg.PluginEnvOverrides, plugins)
 
-	if cfg.PluginEnvOverrides != nil {
```

Comment:
Not sure why the nil check was there; we use range in the next line so nils are not a problem.